docs/source/gcloud_topo.rst

```diff
@@ -8,14 +8,51 @@ Gcloud Topologies
 Google Cloud Topologies
 ```````````````````````
 
-Google cloud topologies . 
+.. code-block:: bash
```

Comment:
@herlo : my bad , I maintained code block yaml through out . Just missed out in gcloud topology.